Source/Scripts/setup-upload-symbols-ios.sh

```diff
@@ -54,27 +57,33 @@ cp $scriptsPath/.bugsplat.conf $HOME/.bugsplat.conf
 
 echo "BugSplat postprocessing: Set actual user credentials"
 
-export bugSplatUser=$(awk -F "=" '/BugSplatUser/ {print $2}' ${configPath}/DefaultEngine.ini)
-export bugSplatPassword=$(awk -F "=" '/BugSplatPassword/ {print $2}' ${configPath}/DefaultEngine.ini)
+export bugSplatDatabase=$(awk -F "=" '/BugSplatDatabase/ {print $2}' ${configPath}/DefaultEngine.ini)
+export bugSplatClientId=$(awk -F "=" '/BugSplatClientId/ {print $2}' ${configPath}/DefaultEngine.ini)
+export bugSplatClientSecret=$(awk -F "=" '/BugSplatClientSecret/ {print $2}' ${configPath}/DefaultEngine.ini)
+
+if [ -z "$bugSplatDatabase" ]; then
+    echo "BugSplat postprocessing: bugSplatDatabase variable is empty"
+fi
 
-if [ -z "$bugSplatUser" ]; then
-    echo "BugSplat postprocessing: bugSplatUser variable is empty"
+if [ -z "$bugSplatClientId" ]; then
+    echo "BugSplat postprocessing: bugSplatClientId variable is empty"
 fi
 
-if [ -z "$bugSplatPassword" ]; then
-    echo "BugSplat postprocessing: bugSplatPassword variable is empty"
+if [ -z "$bugSplatClientSecret" ]; then
+    echo "BugSplat postprocessing: bugSplatClientSecret variable is empty"
 fi
 
-sed -i .backup 's/username/'$bugSplatUser'/g' $HOME/.bugsplat.conf
-sed -i .backup 's/password/'$bugSplatPassword'/g' $HOME/.bugsplat.conf
+sed -i .backup 's/database/'$bugSplatDatabase'/g' $HOME/.bugsplat.conf
+sed -i .backup 's/clientId/'$bugSplatClientId'/g' $HOME/.bugsplat.conf
+sed -i .backup 's/clientSecret/'$bugSplatClientSecret'/g' $HOME/.bugsplat.conf
```

Comment:
I ran into an issue where sed couldn't replace clientSecret in my `.bugsplat.conf` file. After a bit of digging and debug logging I found the issues:

```sh
UATHelper: Packaging (IOS): BugSplat clientSecret: s61/5EPdwWb73skWaiGx2Evs0GiNZLo4
UATHelper: Packaging (IOS): sed: 1: "s/clientSecret/s61/5EPd ...": bad flag in substitute command: 'E'
```

Not sure how to address this... Is there a way to escape the character that's causing the issue?

Reply:
If you get stuck here please let us know, it might be possible for us to just not include `/` in the list of chars used to generate secrets, but I'm not sure.

Reply:
I've added `/` character escaping for the client secret and now script seems to work properly. Please give it another shot and let me know if it helps

Reply:
Looks good 👍